src/modules/roc_sndio/target_sox/roc_sndio/sox_backend.cpp

```diff
@@ -196,5 +196,36 @@ ISource* SoxBackend::open_source(core::IAllocator& allocator,
     return source.release();
 }
 
+void SoxBackend::get_drivers(core::Array<DriverInfo>& arr) {
+    const sox_format_tab_t* formats = sox_get_format_fns();
+    char const* const* format_names;
+    size_t n;
```

Comment:
Please, do not use uninitialized variable and put its initialization inside the for-loop.